lib/screens/onboarding/login/login_screen.dart

```diff
@@ -96,12 +97,20 @@ class LoginScreen extends StatelessWidget {
                 Row(
                   mainAxisAlignment: MainAxisAlignment.end,
                   children: <Widget>[
-                    Text('Forgot password?',
```

Comment:
Just wrap this widget with a GestureDetector. There is an onTap property that comes with it. You can use it to make it do something on tap.